lib/elixir/pages/Typespecs.md

```diff
@@ -35,9 +35,9 @@ The syntax Elixir provides for type specifications is similar to [the one in Erl
                                     ## Numbers
```

Comment:
`float` and `integer` can go.
My idea was that since it's using abbreviations for negative and positive, it wouldn't hurt to be explicit and explain in English words what it is, regardless how obvious it seems.
but it's up to you